parser/parser.go

```diff
@@ -9,9 +9,19 @@ import (
 	"log"
 )
 
+const (
```

Comment:
if we just use the modelfile we can omit all of this

Reply:
These are still useful for consistency. In fact, it would be nice to change command into an interface with each subcommand implementing the interface